ios/NaverLogin.swift

```diff
@@ -16,20 +16,28 @@ class NaverLogin: NSObject, RCTBridgeModule, NaverThirdPartyLoginConnectionDeleg
     static func requiresMainQueueSetup() -> Bool {
         return false
     }
-
-    @objc(login:resolver:rejecter:)
-    func login(options: Dictionary<String, String>, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
-        loginPromiseResolve = resolve
-        loginPromiseReject = reject
-        
+    
+    @objc(initialize:)
+    func initialize(options: Dictionary<String, String>) {
         if let connection = NaverThirdPartyLoginConnection.getSharedInstance() {
             connection.consumerKey = options["consumerKey"]
             connection.consumerSecret = options["consumerSecret"]
             connection.appName = options["appName"]
             connection.serviceUrlScheme = options["serviceUrlScheme"]
             
             connection.delegate = self
-            
+        }
+    }
+
+    @objc(login:rejecter:)
+    func login(resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
+        loginPromiseResolve = resolve
+        loginPromiseReject = reject
+        logoutPromiseResolve = nil
+        logoutPromiseReject = nil
```

Comment:
We have to make null other callback. If not, it make error when we unlink and link again. (old callbacks are called. So, we have to make `nil` them)